docs/guide/audit-presentation.md

```diff
@@ -144,7 +144,16 @@ echo $audit->getModified(true);
 
 **Output:**
```

Comment:
Hi, if you look at this, `output`, means the exact output of `echo $audit->getModified(true);`
If somebody do `echo $audit->getModified(true);`, will not get the formatted json

https://github.com/owen-it/laravel-auditing.com/blob/0e4816384d981b2b10bf9c4f0c3095eeea2cf7c0/docs/guide/audit-presentation.md?plain=1#L141-L148

If you want a formatted json, you have to add `JSON_PRETTY_PRINT` to second parameter, 
but it would not be necessary, since this observation was only to show that it can be converted into json by parameters, you can see the detailed result above, look

https://github.com/owen-it/laravel-auditing.com/blob/0e4816384d981b2b10bf9c4f0c3095eeea2cf7c0/docs/guide/audit-presentation.md?plain=1#L115-L137

Reply:
Okay noted. I understand. Would it help then to perhaps mention that the `getModified()` method also allows for `JSON_PRETTY_PRINT` as the 2nd parameter on this docs page similar to what was done for the `getMetadata()` method or should users assume that's the case? I only ask because I guess it wasn't that obvious to me at first glance when reading the docs until you pointed it out.

Reply:
Normally json should be compressed for performance, so far I have not seen anyone ask for or need `JSON_PRETTY_PRINT`, it was put in for consistency,
 
There is also a third argument: https://www.php.net/manual/en/function.json-encode.php
https://github.com/owen-it/laravel-auditing/blob/22682dd5c800aab90ef261530098625e26a66709/src/Contracts/Audit.php#L63-L74

Reply:
Okay noted. 👍🏼